test/userRegistrations.test.js

```diff
@@ -29,20 +29,65 @@ describe('User Registrations', ()=>{
   const findPendingUsersByEmail = async(email) => (await getPendingUsers()).filter(user => user.login_ids.find(k => k.id === email));
   const findPendingUsersByID = async(id) => (await getPendingUsers()).filter(user => user._id === id);
 
+  const FUNC_NAME = 'confirmFunc';
+  const FUNC_SOURCE = 'exports = function() { return { "status": "success" } }';
+  const createTestFunction = () => ({ name: FUNC_NAME, source: FUNC_SOURCE });
+
   it('removePendingUserByEmail should remove existing pending user', async() => {
     await client.register(testEmail, password);
     expect(await findPendingUsersByEmail(testEmail)).toHaveLength(1);
     await th.app().userRegistrations().removePendingUserByEmail(testEmail);
     expect(await findPendingUsersByEmail(testEmail)).toHaveLength(0);
-    expect(await getPendingUsers()).toHaveLength(1);
+    expect(await getPendingUsers()).toHaveLength(0);
   });
   it('removePendingUserByID should remove existing pending user', async() => {
     await client.register(testEmail, password);
     const userID = (await getPendingUsers())[0]._id;
     expect(await findPendingUsersByID(userID)).toHaveLength(1);
     await th.app().userRegistrations().removePendingUserByID(userID);
     expect(await findPendingUsersByID(userID)).toHaveLength(0);
```

Comment:
can some of these be done inside a `beforeEach`? seems like a bunch of tests do this stuff